dexcom/event_test.go

```diff
@@ -64,38 +45,6 @@ var _ = Describe("Event", func() {
 		Expect(dexcom.EventValueExerciseMinutesMinimum).To(Equal(0.0))
 	})
 
-	It("EventSubTypeHealthAlcohol is expected", func() {
```

Comment:
have combined these into the one test below for the sake of brevity and sanity ... imo anyway